scipy/special/error.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+
+namespace scipy {
+    namespace special {
+#ifndef SP_SPECFUN_ERROR
+        void set_error(const char *func_name, int code, const char *fmt, ...) {
```

Comment:
I think the definition after `#ifndef SP_SPECFUN_ERROR` needs to be inline or else it will break later. The second one doesn't need inline.

Reply:
OK, I had them both inline originally, but got a symbol not found error when importing `lambertw` when `SP_SPECFUN_ERROR` was set. I reflexively took the `inline` out of both, but have confirmed that it still works locally when `SP_SPECFUN_ERROR` is not set and the do nothing `set_error` is inline.